web/src/styles.css

```
@@ -50,6 +50,12 @@ button { color: inherit; }
 .gateway-button { margin-left: auto; min-width: 154px; height: 38px; padding: 0 12px; border: 1px solid var(--line); border-radius: 11px; background: #fff; display: grid; grid-template-columns: 9px 1fr auto; align-items: center; gap: 8px; text-align: left; font-size: 12px; cursor: pointer; box-shadow: 0 1px 2px rgba(0,0,0,.025); }
 .gateway-button > span { width: 8px; height: 8px; border-radius: 50%; background: var(--red); box-shadow: 0 0 0 3px rgba(255,59,48,.09); }
 .gateway-button.live > span { background: var(--green); box-shadow: 0 0 0 3px rgba(52,199,89,.1); }
+
+/* Full width and above the content, because it qualifies everything below it:
+   while it is up, nothing on the page is being refreshed. */
+.live-notice { display: flex; flex-wrap: wrap; align-items: center; justify-content: center; gap: 10px; padding: 8px 24px; border-bottom: 1px solid rgba(191,116,20,.2); background: rgba(255,250,241,.96); color: #89530f; font-size: 11px; }
+.live-notice[data-state="stopped"] { border-bottom-color: rgba(255,59,48,.22); background: rgba(255,241,240,.96); color: #a6302a; }
+.live-notice button { min-height: 26px; padding: 0 10px; border: 1px solid currentColor; border-radius: 8px; background: transparent; color: inherit; font-size: 10px; font-weight: 620; cursor: pointer; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)